app/src/models/uncommitted-changes-strategy.ts

```diff
@@ -1,5 +1,15 @@
-export enum UncommittedChangesStrategy {
+import { IStashEntry } from './stash-entry'
+
+export enum UncommittedChangesStrategyKind {
   askForConfirmation,
   stashOnCurrentBranch,
   moveToNewBranch,
 }
+
+export type UncommittedChangesStrategy =
```

Comment:
These type names are descriptive, but super long. I'm open to suggestions.

Reply:
I'm not too worried about this, but will keep this in mind when doing a proper review